.gitignore

```diff
@@ -4,14 +4,6 @@ test-project-host-*
 dist/
 dist-schema/
 
-# Yarn
```

Comment:
These should stay for now because if you go back to an LTS branch these will end up existing. If we want to create a TODO for their removal we can probably do it in about a year.